examples/case_studies/binning.myst.md

```diff
@@ -10,7 +10,7 @@ kernelspec:
   name: conda-env-pymc_env-py
 ---
 
-(awkward_binning)=
+(binning)=
```

Comment:
we should keep both anchors, I don't know if the original one is already being used somewhere. We did something similar in https://github.com/pymc-devs/pymc-examples/blob/main/examples/generalized_linear_models/multilevel_modeling.ipynb when we combined two notebooks in one.

Reply:
I did it anyway 😂